server/repository/src/migrations/v2_05_00/diagnosis_add_to_invoice.rs

```diff
@@ -14,15 +14,7 @@ impl MigrationFragment for Migrate {
             ALTER TABLE invoice ADD COLUMN diagnosis_id TEXT REFERENCES diagnosis(id);
         "#
         )?;
-
-        // Retranslate all invoices on the next sync, to make sure we capture the diagnosis_id if populated.
-        sql!(
-            connection,
-            r#"
-            UPDATE sync_buffer SET integration_datetime = NULL WHERE table_name = 'invoice';
```

Comment:
invoice table name in sync buffer is 'transact' - so this migration wasn't active in the first place 

(Probably a good thing as we would have risked corrupting data)